data/GESIS-Notebooks.yaml

```diff
@@ -0,0 +1,32 @@
+title: GESIS Notebooks
+provider: GESIS – Leibniz Institute for the Social Sciences
+service_url: http://notebooks.gesis.org
+support: notebooks@gesis.org
+health_api_url: http://notebooks.gesis.org/binder/health
+documentation_url: https://github.com/gesiscss/orc
+target_group_open_for: Open, wich a focus on Computational Social Science
+login_process: Open
+features:
+  version: JupyterHub 3.0.0-beta.1, BinderHub
+  kernels: ["configurable by the user"]
+  extensions: ["configurable by the user"]
```

Comment:
```
kernels: ["configurable by the user"]
extensions: ["configurable by the user"]
```
These fields will be used to filter the services, they should be as specific as possible. Is every programming language / kernel possible with Binder? 
Maybe you can give the pre-installed extensions? That a user can install further stuff afterwards is covered with the "install" key.

Reply:
> Is every programming language / kernel possible with Binder?

Yes

> Maybe you can give the pre-installed extensions?
All handled by the user-side config